lispusers/MODERNIZE

```diff
@@ -325,21 +325,46 @@
                       (IGREATERP LASTMOUSEX (IPLUS LEFT (TIMES WIDTH (DIFFERENCE 1 TITLEPROPORTION])
 
 (NEARESTCORNER
-  [LAMBDA (REGION)                                      (* ; "Edited 14-Feb-2021 21:46 by rmk:")
-
-    (* ;; 
-"Moves the cursor to the corner of REGION that is closest to the current LASTMOUSEX AND LASTMOUSEY")
-
-    (\CURSORPOSITION (CL:IF (ILESSP (IDIFFERENCE LASTMOUSEX (FETCH LEFT OF REGION))
-                                   (IDIFFERENCE (FETCH RIGHT OF REGION)
-                                          LASTMOUSEX))
-                         (FETCH LEFT OF REGION)
-                         (FETCH RIGHT OF REGION))
-           (CL:IF (ILESSP (IDIFFERENCE LASTMOUSEY (FETCH BOTTOM OF REGION))
-                         (IDIFFERENCE (FETCH TOP OF REGION)
-                                LASTMOUSEY))
-               (FETCH BOTTOM OF REGION)
-               (FETCH TOP OF REGION))])
+  [LAMBDA (REGION)                                           (* ; "Edited 29-Jul-2023 10:32 by rmk")
+                                                            (* ; "Edited 14-Feb-2021 21:46 by rmk:")
+
+    (* ;; "Moves the cursor to the corner of REGION that is closest to the current LASTMOUSEX and LASTMOUSEY, provided that that corner is on-screen.")
+
+    (LET ((LEFT (FETCH (REGION LEFT) OF REGION))
+          (RIGHT (FETCH (REGION RIGHT) OF REGION))
+          (TOP (FETCH (REGION TOP) OF REGION))
+          (BOTTOM (FETCH (REGION BOTTOM) OF REGION))
+          X Y)
+
+         (* ;; "If the nearest corner is offscreen, pick the other one.")
+
+         (SETQ X (if (OR (ILESSP LEFT 0)
+                         (IGREATERP LEFT SCREENWIDTH))
```

Comment:
is there an off-by-one here?  shouldn't LEFT > SCREENWIDTH be LEFT >= SCREENWIDTH (or SCREENWIDTH > LEFT)?